Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "influx-replicator"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.72"
+async-stream = "0.3.5"
+bytes = "1.4.0"
+chrono = "0.4.26"
+clokwerk = "0.4.0"
+futures = "0.3.28"
+fallible-iterator = "0.2.0"
+influxdb2-structmap = { git = "https://github.com/foundation-zero/influxdb2.git" }
+itertools = "0.11.0"
+tokio = { version = "1.29.1", features = ["rt", "rt-multi-thread", "macros"], default-features = false }
+tokio-stream = "0.1.14"
+thiserror = "1.0.45"
+clap = { version = "4.3.21", features = ["derive", "env"] }
+env_logger = "0.10.0"
+log = "0.4.20"
+influxdb2 = { git = "https://github.com/foundation-zero/influxdb2.git", features = ["rustls"], default-features = false }
```

Comment:
With https://github.com/aprimadi/influxdb2/pull/40#event-10137396048 having been merged, we can switch this over to that repo. Note that it hasn't been released yet, so it'll still be a git dependency